app/_kong_plugins/acl/index.md

```diff
@@ -25,10 +25,24 @@ topologies:
     - hybrid
     - cloud-gateways
     - serverless
+
 icon: acl.png
 
 categories:
   - traffic-control
+
+search_aliases:
+  - access control list
 ---
 
 ## Overview
+
+The ACL (access control list) plugin allows you to restrict [Consumer](/gateway/entities/consumer/) access to a [Gateway Service](/gateway/entities/service/) or [Route](/gateway/entities/route/). You do this by configuring **either** an allow list or a deny list with certain Consumers<!-- if_version gte:3.7 --> or [Consumer Groups](/gateway/entities/consumer-group/)<!-- endif_version -->.
```

Comment:
```suggestion
The ACL (access control list) plugin allows you to restrict [Consumer](/gateway/entities/consumer/) access to a [Gateway Service](/gateway/entities/service/) or [Route](/gateway/entities/route/). You do this by configuring **either** an allow list or a deny list with certain Consumers or [Consumer Groups](/gateway/entities/consumer-group/).
```
Remove this since the page is evergreen.